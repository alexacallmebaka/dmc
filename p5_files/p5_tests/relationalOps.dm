a1:int = 0;
a2:int = 1;
b:bool;
c:bool = true;
main : () void {
  b = a1 < a2;
  b = a1 < c;
  b = b <= c;
  b = "true" > false;
  b = a1 >= false;
}
