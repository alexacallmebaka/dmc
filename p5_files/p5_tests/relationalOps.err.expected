FATAL [7,12]-[7,13]: Relational operator applied to non-numeric operand
FATAL [8,7]-[8,8]: Relational operator applied to non-numeric operand
FATAL [8,12]-[8,13]: Relational operator applied to non-numeric operand
FATAL [9,7]-[9,13]: Relational operator applied to non-numeric operand
FATAL [9,16]-[9,21]: Relational operator applied to non-numeric operand
FATAL [10,13]-[10,18]: Relational operator applied to non-numeric operand
Type Analysis Failed
